### ISSUE TITLE
problem when scraping series/episode data

### ISSUE THREAD

**Issue by lord-aceldama** [FabianBeiner/PHP-IMDB-Grabber#141]
example 1: Savage Builds
overview page: https://www.imdb.com/title/tt10233836/
episode page: https://www.imdb.com/title/tt10234436/

example 2: The PJs
overview page: https://www.imdb.com/title/tt0182621/
episode page: https://www.imdb.com/title/tt0825938/

**Comment by bla0r** [comment]
can you describe the problem?

**Comment by lord-aceldama** [comment]
i see some of the issues have been fixed in the latest commit. however, with the PJs episodes there still exists a problem when getting the description. in `tt0825938` the description should be `Thurgood catches truant Calvin and puts him to work.` but what is scraped is `The PJs (TV Series) Hangin' with Mr. Super (1999) if ('csm' in window) { csm.measure('csm_body_delivery_started'); } Reference View | Change View Animation, Comedy | 04 Jul 1999 | TV Episode | 7.2 (37) 0 Rate 1 Rate 2 Rate 3 Rate 4 Rate 5 Rate 6 Rate 7 Rate 8 Rate 9 Rate 10 Rate 0 Error: please try again. | Season 1 | Episode 1 | All Episodes (43) | Next › | Thurgood catches truant Calvin and puts him to work.`.

The title is also incorrectly scraped as the series name `The PJs`, not `Hangin' with Mr. Super`. 

getAll() for PJs (tt0825938):
```
Array
(
    [getAka] => Array
        (
            [name] => Aka
            [value] => n/A
        )

    [getAkas] => Array
        (
            [name] => Akas
            [value] => n/A
        )

    [getAspectRatio] => Array
        (
            [name] => AspectRatio
            [value] => n/A
        )

    [getAwards] => Array
        (
            [name] => Awards
            [value] => n/A
        )

    [getBudget] => Array
        (
            [name] => Budget
            [value] => n/A
        )

    [getCast] => Array
        (
            [name] => Cast
            [value] => Eddie Murphy / Loretta Devine / Michael Paul Chan / Cassi Davis / Ja'net DuBois / Cheryl Francis Harrington / Shawn Michael Howard / Jenifer Lewis / Michele Morgan / Kevin Michael Richardson / Pepe Serna / Crystal Scales / Marc Wilmore / Mark Moseley
        )

    [getCastAndCharacter] => Array
        (
            [name] => CastAndCharacter
            [value] => Eddie Murphy as Thurgoode Orenthal Stubbs / Loretta Devine as Muriel Stubbs / Michael Paul Chan as Jimmy Ho / Cassi Davis as HUD woman / Ja'net DuBois as Mrs. Avery / Cheryl Francis Harrington as Ms. Mambo Garcelle (Haiti Lady) / Shawn Michael Howard as Smokey / Jenifer Lewis as Bebe Ho / Michele Morgan as Juicy Hudson / Kevin Michael Richardson as Mr. Hudson / Rasta Man / Pepe Serna as Mr. Sanchez / Crystal Scales as Calvin / Marc Wilmore as Walter Burkett / Mark Moseley as n/A…
        )

    [getCastAndCharacterAsUrl] => Array
        (
            [name] => CastAndCharacterAsUrl
            [value] => Eddie Murphy as Thurgoode Orenthal Stubbs / Loretta Devine as Muriel Stubbs / Michael Paul Chan as Jimmy Ho / Cassi Davis as HUD woman / Ja'net DuBois as Mrs. Avery / Cheryl Francis Harrington as Ms. Mambo Garcelle (Haiti Lady) / Shawn Michael Howard as Smokey / Jenifer Lewis as Bebe Ho / Michele Morgan as Juicy Hudson / Kevin Michael Richardson as Mr. Hudson / Rasta Man / Pepe Serna as Mr. Sanchez / Crystal Scales as Calvin / Marc Wilmore as Walter Burkett / Mark Moseley as n/A…
        )

    [getCastAsUrl] => Array
        (
            [name] => CastAsUrl
            [value] => Eddie Murphy / Loretta Devine / Michael Paul Chan / Cassi Davis / Ja'net DuBois / Cheryl Francis Harrington / Shawn Michael Howard / Jenifer Lewis / Michele Morgan / Kevin Michael Richardson / Pepe Serna / Crystal Scales / Marc Wilmore / Mark Moseley…
        )

    [getCertification] => Array
        (
            [name] => Certification
            [value] => n/A
        )

    [getColor] => Array
        (
            [name] => Color
            [value] => n/A
        )

    [getCompany] => Array
        (
            [name] => Company
            [value] => Will Vinton Studios / Eddie Murphy Productions / Touchstone Television / Buena Vista Television / Fox Network / Warner Bros. Television
        )

    [getCompanyAsUrl] => Array
        (
            [name] => CompanyAsUrl
            [value] => Will Vinton Studios / Eddie Murphy Productions / Touchstone Television / Buena Vista Television / Fox Network / Warner Bros. Television
        )

    [getCountry] => Array
        (
            [name] => Country
            [value] => n/A
        )

    [getCountryAsUrl] => Array
        (
            [name] => CountryAsUrl
            [value] => n/A
        )

    [getCreator] => Array
        (
            [name] => Creator
            [value] => n/A
        )

    [getCreatorAsUrl] => Array
        (
            [name] => CreatorAsUrl
            [value] => n/A
        )

    [getDescription] => Array
        (
            [name] => Description
            [value] => The PJs (TV Series) Hangin' with Mr. Super (1999) if ('csm' in window) { csm.measure('csm_body_delivery_started'); } Reference View | Change View Animation, Comedy | 04 Jul 1999 | TV Episode | 7.2 (37) 0 Rate 1 Rate 2 Rate 3 Rate 4 Rate 5 Rate 6 Rate 7 Rate 8 Rate 9 Rate 10 Rate 0 Error: please try again. | Season 1 | Episode 1 | All Episodes (43) | Next › | Thurgood catches truant Calvin and puts him to work.
        )

    [getDirector] => Array
        (
            [name] => Director
            [value] => John Payson
        )

    [getDirectorAsUrl] => Array
        (
            [name] => DirectorAsUrl
            [value] => John Payson
        )

    [getGenre] => Array
        (
            [name] => Genre
            [value] => Animation / Comedy
        )

    [getGenreAsUrl] => Array
        (
            [name] => GenreAsUrl
            [value] => Animation / Comedy
        )

    [getGross] => Array
        (
            [name] => Gross
            [value] => n/A
        )

    [getLanguage] => Array
        (
            [name] => Language
            [value] => n/A
        )

    [getLanguageAsUrl] => Array
        (
            [name] => LanguageAsUrl
            [value] => n/A
        )

    [getLocation] => Array
        (
            [name] => Location
            [value] => n/A
        )

    [getLocationAsUrl] => Array
        (
            [name] => LocationAsUrl
            [value] => n/A
        )

    [getLocations] => Array
        (
            [name] => Locations
            [value] => n/A
        )

    [getMetaCritics] => Array
        (
            [name] => MetaCritics
            [value] => n/A
        )

    [getMetaScore] => Array
        (
            [name] => MetaScore
            [value] => n/A / n/A
        )

    [getMpaa] => Array
        (
            [name] => Mpaa
            [value] => n/A
        )

    [getPlot] => Array
        (
            [name] => Plot
            [value] => n/A
        )

    [getPlotKeywords] => Array
        (
            [name] => PlotKeywords
            [value] => n/A
        )

    [getPoster] => Array
        (
            [name] => Poster
            [value] => https://m.media-amazon.com/images/G/01/imdb/images/social/imdb_logo._CB410901634_.png
        )

    [getRating] => Array
        (
            [name] => Rating
            [value] => 7.2
        )

    [getRatingCount] => Array
        (
            [name] => RatingCount
            [value] => 37
        )

    [getReleaseDate] => Array
        (
            [name] => ReleaseDate
            [value] => 04 Jul 1999
        )

    [getReleaseDates] => Array
        (
            [name] => ReleaseDates
            [value] => USA / 10 January 1999 / Spain / 19 June 1999 / UK / 4 July 1999 / Norway / 18 August 2000
        )

    [getRuntime] => Array
        (
            [name] => Runtime
            [value] => n/A
        )

    [getSeasons] => Array
        (
            [name] => Seasons
            [value] => n/A
        )

    [getSeasonsAsUrl] => Array
        (
            [name] => SeasonsAsUrl
            [value] => n/A
        )

    [getSoundMix] => Array
        (
            [name] => SoundMix
            [value] => n/A
        )

    [getTagline] => Array
        (
            [name] => Tagline
            [value] => n/A
        )

    [getTitle] => Array
        (
            [name] => Title
            [value] => The PJs
        )

    [getTrailerAsUrl] => Array
        (
            [name] => TrailerAsUrl
            [value] => n/A
        )

    [getType] => Array
        (
            [name] => Type
            [value] => TV Episode
        )

    [getUrl] => Array
        (
            [name] => Url
            [value] => https://www.imdb.com/title/tt0825938/
        )

    [getUserReview] => Array
        (
            [name] => UserReview
            [value] => n/A
        )

    [getVotes] => Array
        (
            [name] => Votes
            [value] => 37
        )

    [getWriter] => Array
        (
            [name] => Writer
            [value] => Eddie Murphy / Steve Tompkins / Larry Wilmore
        )

    [getWriterAsUrl] => Array
        (
            [name] => WriterAsUrl
            [value] => Eddie Murphy / Steve Tompkins / Larry Wilmore
        )

    [getYear] => Array
        (
            [name] => Year
            [value] => 1999
        )

)
```

**Comment by bla0r** [comment]
thx for the dump, you're right... i'll take a look on it

**Comment by bla0r** [comment]
fixed, try it out #145

**Comment by lord-aceldama** [comment]
excellent. that solved my problem. thanks so much!
